### DISCUSSION TITLE
Stack trace does not show the link to the file

### DISCUSSION THREAD

**Discussion by didlika** [cypress-io/cypress#17073] [Questions and Help]
Is there any settings to enable this behaviour since i saw on cypress website that when error happen, stack trace should contain link that will navigate you to the file and line of code where error happened? It's not happening on my end.

My image:
![image](https://user-images.githubusercontent.com/80490488/123089583-6d730680-d427-11eb-98d3-5ddc92929779.png)

Image from cypress:
![image](https://user-images.githubusercontent.com/80490488/123119955-3eb75900-d444-11eb-8595-249e3c06b1bd.png)

**Comment by LashaJini** [comment]
Difference between those two is that `CypressError` happened because there was *syntax or similar* error in your code, as for `AssertionError`, it means your test is failing, that's all. It clearly states that your html element is missing `data-test-id` attribute with value of `step-1-next-step-button`.

Have you tried clicking on the assertion (on line 32) and checking in console?

**Reply by didlika** [reply]
Hey there, thanks for the response! I guess you didn't understand my question. It's not why the test failed ( I failed it on purpose), the question is why I don't have a link that will open test file in the IDEA and navigate me to the method that failed? I would expect that link on any cypress failed method. Like on the cypress image:
![image](https://user-images.githubusercontent.com/80490488/123542770-c4404f00-d74b-11eb-8c7f-0c60a633eac1.png)
